evac/varcallers.py

```diff
@@ -26,8 +24,10 @@ def mpileup_pipeline(args):
         "-l", regions,
         "-v", "-u", "-t DP,AD", caller_args, "-"]
     log.info("Running command: {}".format(' '.join(CMD)))
-    with subprocess.Popen(CMD,stdin=BAM, stdout=OUTDIR) as proc:
-        proc.wait()
+    with open_(args.bam, 'r') as BAM:
```

Comment:
You want to open this with 'rb' since BAM is a binary format. Also, when you are piping to/from another program without doing any preprocessing, generally you can use 'rb' because the target program will take of decoding the data. 'r' (or 'rt')/'w' (or 'wt') mean "wrap this stream in a buffered reader/writer that will perform unicode decoding/encoding of the bytes."